src/components/ListControls/ListControls.vue

```diff
@@ -1,32 +1,35 @@
 <template>
   <div :class="$style.controls">
     <div>
-      <span v-if="tasksAmount === 0">No items left</span>
-      <span v-else-if="tasksAmount === 1">1 item left</span>
-      <span v-else>{{ tasksAmount }} items left</span>
+      <span v-if="tasksLeft === 0">No tasks left</span>
```

Comment:
I think you can move this calculation into a computed property and create a dynamic title